### PR TITLE
Fix cluster auto create schema bug when retry locally 

### PR DESCRIPTION
**Pull request by LebronAl**
This PR fix these issues:
- Fix cluster auto create schema bug when retry locally
- Remove a useless auto create schema error log in `CMManager`
- Downgrade a info log when performing `executeStatementMayRedirect` in `Session`
- Add a flag in `ioTDB` to make server module  fell cluster module

### CODE REVIEW

**Review by HTHou — APPROVED** [review]

**Review by SteveYurongSu — APPROVED** [review]

File: cluster/src/main/java/org/apache/iotdb/cluster/partition/slot/SlotPartitionTable.java [hunk 1]
```diff
@@ -103,7 +104,7 @@ public static void setSlotStrategy(SlotStrategy slotStrategy) {
   private void init(Collection<Node> nodes) {
     logger.info("Initializing a new partition table");
     nodeRing.addAll(nodes);
-    nodeRing.sort(Comparator.comparingInt(Node::getNodeIdentifier));
```
Comment (mychaow) [inline cluster/src/main/java/org/apache/iotdb/cluster/partition/slot/SlotPartitionTable.java:106]: so, if we sort node by ip:port, it's difficult to support ip or port change.

Comment (wangchao316) [reply cluster/src/main/java/org/apache/iotdb/cluster/partition/slot/SlotPartitionTable.java:106]: yes, If the IP address and port are bound, the modification is difficult.

File: cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java [hunk 2]
```diff
@@ -732,7 +732,10 @@ private boolean createTimeseries(
     }
     if (result.getCode() != TSStatusCode.SUCCESS_STATUS.getStatusCode()
         && result.getCode() != TSStatusCode.PATH_ALREADY_EXIST_ERROR.getStatusCode()
-        && result.getCode() != TSStatusCode.NEED_REDIRECTION.getStatusCode()) {
+        && result.getCode() != TSStatusCode.NEED_REDIRECTION.getStatusCode()
+        && result.getCode() == TSStatusCode.MULTIPLE_ERROR.getStatusCode()
+        && result.getSubStatus().stream()
+            .allMatch(s -> s.getCode() == TSStatusCode.PATH_ALREADY_EXIST_ERROR.getStatusCode())) {
```
[the remark this answers is not in the document]
Comment (LebronAl) [reply cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java:738]: You are right! I will fix this. Thanks for your detailed review.

File: cluster/src/main/java/org/apache/iotdb/cluster/partition/slot/SlotPartitionTable.java (hunk 1 above)
Comment (LebronAl) [reply cluster/src/main/java/org/apache/iotdb/cluster/partition/slot/SlotPartitionTable.java:106]: OK, I will remove this.

**Review by HTHou — APPROVED** [review]

**Pull request merged**